packages/lesswrong/components/spotlights/SpotlightItem.tsx

```diff
@@ -49,16 +49,13 @@ const styles = (theme: ThemeType) => ({
     marginBottom: 12,
     boxShadow: theme.palette.boxShadow.default,
     overflow: "hidden", // prevent background image from overflowing if we get styling mismatches in the future, since it depends on the exact height of the review SingleLineComment
-    // TODO these were added to fix an urgent bug, hence the forum gating. Maybe they could be un-gated
-    ...(isFriendlyUI && {
-      maxWidth: SECTION_WIDTH,
-      marginLeft: "auto",
-      marginRight: "auto",
-      [theme.breakpoints.up('md')]: {
-        width: SECTION_WIDTH // TODO: replace this hacky solution with a more comprehensive refactoring of SingleColumnSection.
-        // (SingleColumnLayout should probably be replaced by grid-css in Layout.tsx)
-      }
-    })
+    maxWidth: SECTION_WIDTH,
+    marginLeft: "auto",
+    marginRight: "auto",
+    [theme.breakpoints.up('md')]: {
+      width: SECTION_WIDTH // TODO: replace this hacky solution with a more comprehensive refactoring of SingleColumnSection.
+      // (SingleColumnLayout should probably be replaced by grid-css in Layout.tsx)
+    }
```

Comment:
Is this supposed to be ungated?

Reply:
It actually caused me problems in some edge-cases when it didn't exist.